src/rattler_build_conda_compat/recipe_sources.py

```diff
@@ -138,7 +138,7 @@ def render(template: str | list[str], context: dict[str, str]) -> str | list[str
 
             context = recipe.get("context", {})
             # render out the context section and retrieve dictionary
-            context_variables = load_recipe_context(context, env)
```

Comment:
This means that the variant variables are passed as `context` to the Source object below. Is this correct?